### DISCUSSION TITLE
Problem with socket after creating an exe

### DISCUSSION THREAD

**Discussion by Jey0707** [miguelgrinberg/python-socketio#994] [Q&A]
Hi, I'm having a problem after I create an exe with socket io using auto-py-to-exe.

When I start the .exe, the server in localhost is not started immediatly but with some delay and have some problems on reciving/sending message, I don't know how to log more information about it.

![image](https://user-images.githubusercontent.com/86539443/183062086-c3542ecd-7c20-4ad2-a7fa-d44d579da336.png)

**Comment by miguelgrinberg** [comment]
I can't really help with exe issues as I don't use those tools myself. If the application works well when running normally, then I suggest you search for other discussions here regarding exe generation, where people have been able to get this to work. Unfortunately I'm not familiar with the details.

**Comment by zhenzi0322** [comment]
> Hi, I'm having a problem after I create an exe with socket io using auto-py-to-exe.
> 
> When I start the .exe, the server in localhost is not started immediatly but with some delay and have some problems on reciving/sending message, I don't know how to log more information about it.
> 
> ![image](https://user-images.githubusercontent.com/86539443/183062086-c3542ecd-7c20-4ad2-a7fa-d44d579da336.png)

When an exception occurs, you have to make sure that the server side starts first, and finally the client side starts
